src/shell_minimal_runtime.html

```diff
@@ -2,174 +2,19 @@
 <body>
 <canvas style='display:block; margin:auto;'></canvas>
 <script>
-  function binary(url) { // Downloads a binary file and outputs it in the specified callback
-    return new Promise((ok, err) => {
-      var x = new XMLHttpRequest();
-      x.open('GET', url, true);
-      x.responseType = 'arraybuffer';
-      x.onload = () => { ok(x.response); }
-      x.send(null);
-    });
-  }
-
```

Comment:
Adding `-s WASM=2` support to MINIMAL_RUNTIME started to make the minimal runtime loader shell quite cluttered, so migrated this whole thing to be dynamically generated instead from python.